lol_alloc/src/free_list_allocator.rs

```diff
@@ -173,14 +173,13 @@ fn full_size(layout: Layout) -> usize {
 // From https://github.com/wackywendell/basicalloc/blob/0ad35d6308f70996f5a29b75381917f4cbfd9aef/src/allocators.rs
 // Round up value to the nearest multiple of increment
 fn round_up(value: usize, increment: usize) -> usize {
-    if value == 0 {
-        return 0;
-    }
-    increment * ((value - 1) / increment + 1)
+    debug_assert!(increment.is_power_of_two());
+    (value + (increment - 1)) & increment.wrapping_neg()
```

Comment:
At least to me its less clear why this works now, so a comment here clarifying it would be nice.